main.py

```diff
@@ -48,6 +48,7 @@ def __init__(self):
 		self.clock = pygame.time.Clock()
 		pygame.key.set_repeat(1, 100)
 		self.graphics = assets.populate_assets()
+		self.itemslot = 1
```

Comment:
what the fuck doet itemslot in de fucking ENGINE. De Game class is de ENGINE. Daar ga je niet shit in stoppen wat alleen de PLAYER nodig heeft. Als je code LEEST, kan je zien dat in de PLAYER CLASS echt al ongeveer 5 MAANDEN een begin van een inventory system staat, dat op het moment werkt met 2 HANDEN ZOALS WE DAT BESPROKEN HEBBEN IN DE DISCORD 5 MAANDEN GELEDEN. Maar ik vind t goed als we het zo doen, daar gaat het mij niet om. Het gaat mij er vooral om dat je leert CODE TE LEZEN EN NIET DEZE BULLSHIT ZO TE SCHRIJVEN. IK KAN HET OOK ZELF SCHRIJVEN, WAT IK NU EFFECTIEF DOE.....................
Nogmaals, ik wil best het equipment system herschrijven voor andere functionaliteit. Ik geef geen fuck. Maar snap mijn punt eens. Dit is alles wat je hebt kunnen doen in een fucking week en ik moet 90% nog gaan herschrijven/zeggen dat je het moet verbeteren om dat je gewoon te LUI bent om al bestaande code te lezen om te snappen hoe het allemaal in elkaar zit?? En ben je ook te lui om vragen te stellen in de discord of ben je bang dat je afgezeikt wordt. Wat het ook is, ik moedig iedereen al aan om vragen te stellen. Ik heb toch ook vaak openlijk gesteld hoe ik iets heb geschreven en of jullie veranderingen willen?